view/frontend/templates/js/replacejs.phtml

```diff
@@ -1017,6 +994,7 @@ ob_start();
         };
         ///////////////////////////////////////////////////////////
 
+        if (getCheckoutType()=="payment") {
```

Comment:
Same here.

Reply:
The same question. How can I emulate this?

Reply:
Do I understand correctly? In multistep we want to use only three fields for hints: country, State/Province and Zip code?

Reply:
![image](https://user-images.githubusercontent.com/7270718/84755378-4723bb00-afca-11ea-8ec5-cfa2d5f55cc7.png)

Reply:
@vitaliyreznikov those 3 fields are enough to get the shipping pre-fetch / estimation. In many cases the actual street address does not make a difference. Even the state is not mandatory, not all countries have states or regions. So yes, we call prefetch with whatever data we have available and save the results in cache. However, when prefetch is triggered it does a few more calculations if possible, based on the following
1. address saved to quote, if any
2. logged in customer default address, if any
3. address determined by geolocation service (https://ipstack.com/), if configured.

Reply:
Ok, so let me don't change it for now. Updated.